lib/spack/spack/stage.py

```diff
@@ -49,6 +50,13 @@
 stage_prefix = "spack-stage-"
 
 
+def compute_stage_name(prefix: str, spec):
```

Comment:
The `prefix` parameter isn't used in this method.

I think the parameter should be removed and the `os.path.join` should occur outside of it (which I think already happens in `Stage.__init__`).